### ISSUE TITLE
Missing docstr for `HTTP.request()`

### ISSUE THREAD

**Issue by mattBrzezinski** [JuliaWeb/HTTP.jl#459]
https://juliaweb.github.io/HTTP.jl/stable/public_interface/#Requests-1

```
Missing docstring for HTTP.request(::String,::HTTP.URIs.URI,::Array{Pair{SubString{String},SubString{String}},1},::Any).
Check Documenter's build log for details.
```

**Comment by benelsen** [comment]
As far as I can tell, to fix it someone just needs to merge https://github.com/JuliaWeb/HTTP.jl/pull/446 and do a release or docs rebuild.

**Comment by quinnj** [comment]
Fixed via #446
